test/src/e2e/partialSignature.test.ts

```diff
@@ -341,8 +331,7 @@ describe("Partial signature", function() {
                 });
                 const hash = await node.sendAssetTransaction(tx);
                 expect(await node.sdk.rpc.chain.getTransaction(hash)).not.null;
-                expect(await node.sdk.rpc.chain.containsTransaction(hash)).be
-                    .true;
+                expect(await node.sdk.rpc.chain.containsTransaction(hash)).be.true;
```

Comment:
You shouldn't have changed it.
You reverted this change in https://github.com/CodeChain-io/codechain/pull/1475/commits/455792e7359dae672dc7d942b962700517adb70c

Reply:
I ran `yarn fmt` in every commit not to make such changes.

Reply:
It seems that it's not fixed yet.

Reply:
Now there is no such changes reverted in the middle of the commit in this pr. Sorry for your rechecking.